src/pages/landing/weekend-trading/components/_icon-text-row.js

```diff
@@ -164,11 +166,13 @@ const IconTextRow = () => {
                                 {localize('Trade anytime')}
                             </StyledCardHeader>
                             <StyledText
+                                lh="31.2px"
                                 size="var(--text-size-m)"
```

Comment:
Are you sure line heights are in float values like these? `var(--text-size-m)` is actually deprecated already. Depends should usually given font sizes in pixels for desktop & mobile separately.

Reply:
lh updated to integer, used 24px instead of --text-size-m